Cargo.toml

```diff
@@ -37,6 +37,7 @@ serde_json = "1.0.89"
 schemars = "0.8.10"
 sha3 = { version = "0.10.8", default-features = false, features = [] }
 signature-verifier-api = { version = "^0.1.0", path = "packages/signature-verifier-api" }
+axelar-encoding = { git = "ssh://git@github.com/eigerco/solana-axelar-internal.git", branch = "main"}
```

Comment:
Temporary fetching from private repo, just until we stablise API's

Reply:
Maybe lets rename the crate import here to something like `axelar-rkyv-encoding`? We should probably propagate this rename to the source crate as well.

Reply:
I would be OK with this change, lets ping @tilacog to check with him.

Reply:
Great ! fixed at 23be5851636afac76128f097f0353aa15bc3b7fe